packages/provider.databind/spec/dataBindProviderBehaviors.ts

```diff
@@ -38,6 +38,9 @@ import {
 
 import * as coreBindings from '@tko/binding.core';
 
+import { assert} from "chai"
```

Comment:
these vendor imports should be above the "relative" imports (not a blocker)

Reply:
Done in https://github.com/Auge19/tko/pull/4, i have also fixed unneeded imports and testseries closure. I think everything is only stylistic icing on the cake.